resources/views/student-council/communication-committee/epistola.blade.php

```diff
@@ -8,52 +8,24 @@
         grey lighten-2
         @endif
     @endif
-    ">
+    " >
     <div class="card-image" style="overflow-x: auto;overflow-y: hidden;">
         <img class="materialboxed" id="{{$article->id}}" src="{{ $article->picture_path ?? url('/img/committee-logos/kommbiz.jpg')}}" onerror="standby({{$article->id}})">
     </div>
-    <div class="card-content activator">
-        <i class="material-icons float right" style="cursor: pointer">more_vert</i></span>
+    <div class="card-content" onclick="window.location='{{ route('epistola.preview')}}'"
```

Comment:
I'm not sure if that's the best way for redirecting, but ok.

Reply:
Using links (<a>tags) messed up the ui, that's why I used the Javascript method.